dispatches/models/renewables_case/RE_flowsheet.py

```diff
@@ -0,0 +1,368 @@
+                                              MomentumMixingType,
+                                              Valve,
+                                              ValveFunctionType)
+
+from dispatches.models.nuclear_case.properties.h2_ideal_vap \
+    import configuration as h2_ideal_config
+from dispatches.models.nuclear_case.properties.hturbine_ideal_vap \
+    import configuration as hturbine_config
+import dispatches.models.nuclear_case.properties.h2_reaction \
+    as h2_reaction_props
+
+from idaes.generic_models.unit_models.product import Product
+from idaes.generic_models.unit_models.separator import Separator
+from dispatches.models.nuclear_case.unit_models.hydrogen_turbine_unit import HydrogenTurbine
+from dispatches.models.nuclear_case.unit_models.hydrogen_tank import HydrogenTank
+from dispatches.models.nuclear_case.unit_models.hydrogen_tank_simplified import SimpleHydrogenTank as SimpleHydrogenTank
+from dispatches.models.renewables_case.load_parameters import *
+from dispatches.models.renewables_case.pem_electrolyzer import PEM_Electrolyzer
+from dispatches.models.renewables_case.elec_splitter import ElectricalSplitter
+from dispatches.models.renewables_case.battery import BatteryStorage
+from dispatches.models.renewables_case.wind_power import Wind_Power
+
+timestep_hrs = 1
+H2_mass = 2.016 / 1000
+
+PEM_temp = 300
+H2_turb_pressure_bar = 24.7
+max_pressure_bar = 700
+
+
+def add_wind(m, wind_mw, wind_config=None):
+    resource_timeseries = dict()
+    for time in list(m.fs.config.time.data()):
+        # ((wind m/s, wind degrees from north clockwise, probability), )
+        resource_timeseries[time] = ((10, 180, 0.5),
+                                     (24, 180, 0.5))
+    if wind_config is None:
+        wind_config = {'resource_probability_density': resource_timeseries}
+
+    m.fs.windpower = Wind_Power(default=wind_config)
+    m.fs.windpower.system_capacity.fix(wind_mw * 1e3)   # kW
+    return m.fs.windpower
+
+
+def add_pem(m, outlet_pressure_bar):
+    m.fs.h2ideal_props = GenericParameterBlock(default=h2_ideal_config)
+    m.fs.h2ideal_props.set_default_scaling('flow_mol_phase', 1)
+    m.fs.h2ideal_props.set_default_scaling('mole_frac_comp', 1)
+    m.fs.h2ideal_props.set_default_scaling('mole_frac_phase_comp', 1)
+    m.fs.h2ideal_props.set_default_scaling('flow_mol', 1)
+    m.fs.h2ideal_props.set_default_scaling('enth_mol_phase', 0.1)
+
+    m.fs.pem = PEM_Electrolyzer(
+        default={"property_package": m.fs.h2ideal_props})
+
+    # Conversion of kW to mol/sec of H2. (elec*elec_to_mol) based on H-tec design of 54.517kW-hr/kg
+    m.fs.pem.electricity_to_mol.fix(0.002527406)
+    m.fs.pem.outlet.pressure.setub(max_pressure_bar * 1e5)
+    m.fs.pem.outlet.pressure.fix(outlet_pressure_bar * 1e5)
+    m.fs.pem.outlet.temperature.fix(PEM_temp)
+    return m.fs.pem, m.fs.h2ideal_props
+
+
+def add_battery(m, batt_mw):
+    m.fs.battery = BatteryStorage()
+    m.fs.battery.charging_eta.set_value(1)
```

Comment:
Seems like the charging and discharging efficiency are fixed at 1 here.

Reply:
Changed to 0.98

Reply:
Thanks, @dguittet ! Sorry to bother you again, but could you make this to 0.95 so it is consistent with this: https://github.com/xiangao1/dispatches/blob/a5d85fd8b5d42e7800f20cc5fb957965ae323b1a/dispatches/models/renewables_case/battery.py#L85.

Reply:
No problem, updated!